kiyoh-customerreview.php

```diff
@@ -27,14 +27,18 @@
             }
         }
         add_action("save_post", "check_kiyoh_review", 10, 2);
+        add_action("woocommerce_after_order_object_save", "check_kiyoh_reviewNew", 10, 2);
     }
 }
 
+function check_kiyoh_reviewNew($post, $wc_data_store)

```

Comment:
I'd suggest to use more specific name for the function:
```suggestion
function check_kiyoh_review_on_order_save($post, $wc_data_store)
```